app/jobs/set_default_parent_thumbnail_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SetDefaultParentThumbnailJob < ApplicationJob
+  queue_as :import
+
+  def perform(child_work:, parent_work:)
+    parent_work.reload
+    return if parent_work.thumbnail.is_a?(Attachment)
```

Comment:
we probably want to add a return logic here and only set them if the thumbnail is blank. 

Additionally it may be helpful to add a rake task for us to run this job on the fly. 

Ref LV for examples

rake task: https://github.com/scientist-softserv/louisville-hyku/blob/main/lib/tasks/set_parent_thumbnail.rake
job: https://github.com/scientist-softserv/louisville-hyku/blob/main/app/jobs/set_default_parent_thumbnail_job.rb

Reply:
Yes I was saying if it is not an Attachment (including NilClass) then return.

And also yes, I used all those as inspiration and added the rake task in this PR as well.